<!--REQUEST_ID=1d684794-bcaa-4584-b8a7-73b2ff582cff-->
Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.47979&attempt_id=8057dd7a-f3d3-44ce-9098-1237da29a589)

Attempted: howl

<details><summary>Partial log (click to expand)</summary><p>

```
/nix/store/f2lp21azk4w88f17zgf4mji64agb18ga-bash-4.4-p23/bin/bash: gcc: command not found
/nix/store/f2lp21azk4w88f17zgf4mji64agb18ga-bash-4.4-p23/bin/bash: gcc: command not found
/nix/store/f2lp21azk4w88f17zgf4mji64agb18ga-bash-4.4-p23/bin/bash: gcc: command not found
Makefile:268: *** Unsupported target architecture.  Stop.
make[2]: Leaving directory '/private/tmp/nix-build-howl-0.5.3.drv-0/howl-0.5.3/src/deps/LuaJIT-2.1.0-beta3/src'
make[1]: *** [Makefile:113: default] Error 2
make[1]: Leaving directory '/private/tmp/nix-build-howl-0.5.3.drv-0/howl-0.5.3/src/deps/LuaJIT-2.1.0-beta3'
make: *** [Makefile:58: deps/LuaJIT-2.1.0-beta3/src/libluajit.a] Error 2
builder for '/nix/store/h3fjvcvcwdfm5y09l0xrarmbnzczi5in-howl-0.5.3.drv' failed with exit code 2
error: build of '/nix/store/h3fjvcvcwdfm5y09l0xrarmbnzczi5in-howl-0.5.3.drv' failed
```
</p></details>

--- a/pkgs/applications/editors/howl/default.nix
+++ b/pkgs/applications/editors/howl/default.nix
@@ -1,0 +1,40 @@
+{ stdenv, fetchurl, makeWrapper, pkgconfig, gtk3, librsvg }:
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+  name = "howl-${version}";
+  version = "0.5.3";
+
+  # Use the release tarball containing pre-downloaded dependencies sources
+  src = fetchurl {
+    url = "https://github.com/howl-editor/howl/releases/download/0.5.3/howl-0.5.3.tgz";
+    sha256 = "0gnc8vr5h8mwapbcqc1zr9la62rb633awyqgy8q7pwjpiy85a03v";
+  };
+
+  sourceRoot = "./howl-${version}/src";
+
+  # The Makefile uses "/usr/local" if not explicitly overridden
+  installFlags = [ "PREFIX=$(out)" ];
+
+  nativeBuildInputs = [ makeWrapper pkgconfig ];
+  buildInputs = [ gtk3 librsvg ];
+  enableParallelBuilding = true;
+
+  # Required for the program to properly load its SVG assets
+  postInstall = ''
+    wrapProgram $out/bin/howl \
+      --set GDK_PIXBUF_MODULE_FILE "$GDK_PIXBUF_MODULE_FILE"
+  '';
+
+  meta = {
+    homepage = https://howl.io/;
+    description = "A general purpose, fast and lightweight editor with a keyboard-centric minimalistic user interface";
+    license = licenses.mit;
+    maintainers = with maintainers; [ pacien ];
+
+    # LuaJIT and Howl builds fail for x86_64-darwin and aarch64-linux respectively
+    platforms = [ "i686-linux" "x86_64-linux" ];
+  };
+}
+

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -17022,6 +17022,8 @@ with pkgs;
     gtk = gtk3;
   };
 
+  howl = callPackage ../applications/editors/howl { };
+
   ht = callPackage ../applications/editors/ht { };
 
   hubstaff = callPackage ../applications/misc/hubstaff { };

